version

```diff
@@ -1 +1 @@
-4.1.13
+4.1.14
```

Comment:
As @fepitre noted already, please don't bump version in a PR. The version bump will be required, plus adjusting shlibs file, but state this in the PR description instead of changing the file directly. The reason is simple: we want the "version commit" to be the one that is tagged with a version tag - this won't work if the version is bumped somewhere under a merge commit.